Skate_ERP_preprocessing_IO.m

```diff
@@ -75,326 +66,77 @@
+        tempEEG = EEG;
         
-        
-        EEG = pop_selectevent( tempEEG, 'type',3 ,'renametype','Standard','deleteevents','on','deleteepochs','on','invertepochs','off');
-        EEG = pop_editset(EEG, 'setname',[subs{i_sub} '_' exp '_' conds{i_cond} '_Corrected_Standard']);
-        EEG = pop_saveset( EEG, 'filename',[subs{i_sub} '_' exp '_facing_in_Corrected_Standard.set'],'filepath',[Pathname 'segmentsIO\']);
-        
-        end
-        
-    end 
-end 
-commandhistory
-%%
-
-%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
-%                    REGULAR OUT
-%
-%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
-%%
-clear all
-close all
-ccc
+ %now select the corrected trials
+%         EEG = pop_selectevent( tempEEG, 'type',5,'renametype','Target','deleteevents','on','deleteepochs','on','invertepochs','off');
+%         EEG = pop_editset(EEG, 'setname',[subs{i_sub} '_' exp '_' conds{i_cond} '_Corrected_Target']);
+%         EEG = pop_saveset( EEG, 'filename',[subs{i_sub} '_' exp '_' conds{i_cond} '_Corrected_Target.set'],'filepath',[Pathname 'segmentsIO\']);
+%         
+%         
+%         EEG = pop_selectevent( tempEEG, 'type',3 ,'renametype','Standard','deleteevents','on','deleteepochs','on','invertepochs','off');
+%         EEG = pop_editset(EEG, 'setname',[subs{i_sub} '_' exp '_' conds{i_cond} '_Corrected_Standard']);
+%         EEG = pop_saveset( EEG, 'filename',[subs{i_sub} '_' exp '_' conds{i_cond} '_Corrected_Standard.set'],'filepath',[Pathname 'segmentsIO\']);
 
-exp = 'Skateboard';
-
-% loading regulars first
-subs = {'100' '101' '102' '103' '104' '106' '108' '111'...
-  '112' '116' '117' '118' '119' '120'};
-%subs = {'100'}; %to test on just one sub
+        if is_goofy(i_sub) 
+		
+			%this strcmp is how to compare strings 
+			% || means or, but | may be the right sympol (i always forget)
 
- nsubs = length(subs);
-%conds = {'P_CW';'P_CCW'; 'NP_CW'; 'NP_CCW'};
-conds =  {'P_CCW';'NP_CW'}; %facing OUTSIDE for regulars
-conds2 = {'facing_out1'; 'facing_out2'};
+			if strcmp(conds{i_cond},'P_CCW') || strcmp(conds{i_cond},'NP_CW')
+				ThisIsTheName = [subs{i_sub} '_' exp '_' new_conds{1} '_Corrected_Standard.set']
+            else
+				ThisIsTheName = [subs{i_sub} '_' exp '_' new_conds{2} '_Corrected_Standard.set']
+			end
 
-nconds = length(conds);
-Pathname = 'M:\Data\Skateboard\winter2019\';
 
-if exist([Pathname 'segments\'])
-    mkdir([Pathname 'segmentsIO\']);
-end
-[ALLEEG EEG CURRENTSET ALLCOM] = eeglab;
+		elseif ~is_goofy(i_sub) %if its a zero
 
+			if strcmp(conds{i_cond},'P_CCW') || strcmp(conds{i_cond},'NP_CW')
 
-for i_sub = 1:nsubs
-    for i_cond = 1:nconds
-          
+				ThisIsTheName = [subs{i_sub} '_' exp '_' new_conds{2} '_Corrected_Standard.set']
+				 %notice this is opposite the goofy
+			else
+				ThisIsTheName = [subs{i_sub} '_' exp '_' new_conds{1} '_Corrected_Standard.set']
+            end
```

Comment:
make sure that this end statement matches the if is_goofy and elseif ~is_goofy

Reply:
okay